### PR TITLE
Port WinForms docs for 9.0

### PR DESCRIPTION
**Pull request by lonitra**
## Summary

Fixes https://github.com/dotnet/winforms/issues/11845
Followed porting instructions in main issue

porting documentation for darkmode / async feature still pending

### CODE REVIEW

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ToolStripItem.xml:7285]: Would you mind reinstalling the porting tool so that this does not happen again? Simply sync to the latest bits in main and run the `.\install-as-tool.ps1` from dotnet/api-docs-sync. I fixed the annoying `<para>` bug yesterday.

Also, some fixes here. I think ToolStrip**Menu**item might be wrong in the text, can you confirm?:

```suggestion
          <para>This event is raised when item is selected by mouse, keyboard or programmatically.</para>
          <para>The <c>MainMenu</c> and <c>MenuItem</c> controls were removed in .NET Core 3.1. The <see cref="E:System.Windows.Forms.ToolStripItem.SelectedChanged" /> event is recommended as a replacement for the <c>MenuItem.Select</c> event.</para>
```

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ToolStripItem.xml:5863]: The second and third paragraphs should go in remarks:

```suggestion
        <summary>Raises the <see cref="E:System.Windows.Forms.ToolStripItem.SelectedChanged" /> event.</summary>
```

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ToolStripItem.xml:5864]:
```suggestion
        <remarks>
          <para>This method will be called when selected <see cref="T:System.Windows.Forms.ToolStripItem" /> changes.</para>
          <para>Call <c>base.OnSelectedChanged</c> to send this event to any registered event listeners.</para>
        </remarks>
```

File: xml/System.Windows.Forms/ToolStrip.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ToolStrip.xml:205]: We should not add full API items ourselves. They are added automatically when we feed a new preview or RC version.

I assume this API was added in P7, but right now dotnet-api-docs is showing APIs from P6. Can you please confirm that this API is being added in the new CI update? https://github.com/dotnet/dotnet-api-docs/pull/10257

```suggestion
```

File: xml/System.Windows.Forms/GiveFeedbackEventArgs.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/GiveFeedbackEventArgs.xml:329]:
```suggestion
        <remarks>Specify <see langword="true" /> for <see cref="P:System.Windows.Forms.GiveFeedbackEventArgs.UseDefaultDragImage" /> to use a layered window drag image with a size of 96x96; otherwise <see langword="false" />.</remarks>
```

**Review by carlossanlop — COMMENTED** [review]
Left some comments.

File: xml/System.Windows.Forms/ToolStrip.xml
Comment (lonitra) [reply xml/System.Windows.Forms/ToolStrip.xml:205]: Thanks for the information! I was wondering why this one wasn't showing up. Yes, that PR adds this API among some other System.Drawing ones. I will wait until that PR goes in then and pull in the changes.

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (lonitra) [reply xml/System.Windows.Forms/ToolStripItem.xml:7285]: I believe this was intentional to be ToolStripMenuItem.SelectedChanged @Tanya-Solyanik can you double check if that is correct?

File: xml/System.Windows.Forms/GiveFeedbackEventArgs.xml
Comment (gewarren) [inline xml/System.Windows.Forms/GiveFeedbackEventArgs.xml:325]: Not sure why this is needed - it's just repeating what's in \<value>.

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (gewarren) [inline xml/System.Windows.Forms/ToolStripItem.xml:5860]:
```suggestion
          <para>This method will be called when the selected <see cref="T:System.Windows.Forms.ToolStripItem" /> changes.</para>
```

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (gewarren) [inline xml/System.Windows.Forms/ToolStripItem.xml:7300]:
```suggestion
        <summary>Occurs when the value of the <see cref="P:System.Windows.Forms.ToolStripItem.Selected" /> property changes.</summary>
```

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (gewarren) [inline xml/System.Windows.Forms/ToolStripItem.xml:7302]:
```suggestion
          <para>This event is raised when an item is selected by mouse or keyboard, or programmatically.</para>
```

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (gewarren) [inline xml/System.Windows.Forms/ToolStripItem.xml:7303]:
```suggestion
          <para>.NET Core 3.1 removed the MainMenu and MenuItem controls. The ToolStripMenuItem.SelectedChanged event</para>
```

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (gewarren) [inline xml/System.Windows.Forms/ToolStripItem.xml:7304]:
```suggestion
          <para>is recommended as a replacement for the MenuItem.Select event.</para>
```

**Review by gewarren — APPROVED** [review]

**Comment by lonitra** [comment]
@gewarren do we have an estimate for when the CI update for RC 1 will be? WinForms has some apis that were introduced in RC1 so this is in draft to get those docs ported as well.

**Comment by gewarren** [comment]
> @gewarren do we have an estimate for when the CI update for RC 1 will be? WinForms has some apis that were introduced in RC1 so this is in draft to get those docs ported as well.

@lonitra It's usually a couple of days after it releases.

**Comment by carlossanlop** [comment]
@lonitra so I assume you're going to wait for RC1 to be released before merging this?

**Comment by lonitra** [comment]
> so I assume you're going to wait for RC1 to be released before merging this?

Yes, we need the documentation for the experimental APIs as well to finish up documentation for net 9 so this will need to wait until the CI update.

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (Tanya-Solyanik) [inline xml/System.Windows.Forms/ToolStripItem.xml:7303]:
```suggestion
          <para>.NET Core 3.1 removed the MainMenu and MenuItem controls. The ToolStripItem.SelectedChanged event</para>
```

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (Tanya-Solyanik) [reply xml/System.Windows.Forms/ToolStripItem.xml:7304]: This and the previous suggestion wasn't applied, looks like a bad merge...

Comment (lonitra) [reply xml/System.Windows.Forms/ToolStripItem.xml:7304]: Thanks for the catch!

File: xml/System.Windows.Forms/Form.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/Form.xml:7974]:
```suggestion
        <exception cref="T:System.InvalidOperationException">
          <para>The form is already displayed asynchronously.</para>
          <para>-or-</para>
          <para>No <see cref="T:System.Windows.Forms.WindowsFormsSynchronizationContext" /> could be retrieved or installed.</para>
        </exception>
```

File: xml/System.Windows.Forms/GiveFeedbackEventArgs.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/GiveFeedbackEventArgs.xml:325]: This is the default text, it should stay if it's meant to be empty.
```suggestion
        <remarks>To be added.</remarks>
```

File: xml/System.Windows.Forms/TaskDialog.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TaskDialog.xml:274]:
```suggestion
        <remarks>Showing the dialog will bind the <paramref name="page" /> and its controls until this method returns or the dialog is navigated to a different page.</remarks>
```

File: xml/System.Windows.Forms/TaskDialog.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TaskDialog.xml:308]: Is it correct to say a parameter "gets or sets"?

File: xml/System.Windows.Forms/TaskDialog.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TaskDialog.xml:315]:
```suggestion
        <remarks>Showing the dialog will bind the <paramref name="page" /> and its controls until this method returns or the dialog is navigated to a different page.</remarks>
```

File: xml/System.Windows.Forms/TaskDialog.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TaskDialog.xml:349]: Same question.

File: xml/System.Windows.Forms/TaskDialog.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TaskDialog.xml:356]:
```suggestion
        <remarks>Showing the dialog will bind the <paramref name="page" /> and its controls until this method returns or the dialog is navigated to a different page.</remarks>
```

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ToolStripItem.xml:6688]:
```suggestion
        <summary>Returns the parent <see cref="T:System.Windows.Forms.ToolStrip" />'s renderer.</summary>
```

File: xml/System.Windows.Forms/ToolStripItem.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ToolStripItem.xml:7304]:
```suggestion
          <para>The <c>MainMenu</c> and <c>MenuItem</c> controls were removed in .NET Core 3.1. The <c>ToolStripItem.SelectedChanged</c> event is recommended as a replacement for the <c>MenuItem.Select</c> event.</para>
```

File: xml/System.Windows.Forms/TaskDialog.xml
Comment (lonitra) [inline xml/System.Windows.Forms/TaskDialog.xml:308]:
```suggestion
        <param name="startupLocation">The position of the task dialog when it is shown.</param>
```

File: xml/System.Windows.Forms/TaskDialog.xml
Comment (lonitra) [inline xml/System.Windows.Forms/TaskDialog.xml:349]:
```suggestion
        <param name="startupLocation">The position of the task dialog when it is shown.</param>
```

**Pull request merged**
